subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_uart.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright (c) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/mgmt/ec_host_cmd/backend.h>
+#include <zephyr/mgmt/ec_host_cmd/ec_host_cmd.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/time_units.h>
+#include <zephyr/sys/util.h>
+
+LOG_MODULE_REGISTER(host_cmd_uart, CONFIG_EC_HC_LOG_LEVEL);
+
+/*
+ * Enumeration to maintain different states of incoming request from
+ * host
+ */
```

Comment:
I find this very redundant, the enum name is meaningful enough and it's not like these are going to a generated documentation anyway. I'd say drop it, keep the file a bit more compact.

Reply:
Done.